perf_tests/large_string_join.jsonnet

```diff
@@ -0,0 +1,5493 @@
+{
+    text: std.join(',', [
```

Comment:
Can we avoid having thousands of lines here? Navigating this is pretty annoying...

Reply:
you mean do `std.join(',', std.makeArray(n, function() 'x'))` ?

sure although it'd be interesting to see how performance differs between the two.

Reply:
you made some benchmarks as well, right?

I'd like to have realistic ones if possible.

Reply:
Yeah, something like that.

For truly realistic examples a corpus of real code would be the best, probably.